client/js/components/statement/splitStatement/DpCreateTag.vue

```diff
@@ -254,27 +261,34 @@ export default {
                     }
                   }
                   this.updateTags(tagResource)
+                  this.$root.$emit('tag:created', tagResource.id)
```

Comment:
Can we also watch for `state.availableTags` in the addon, instead of emitting an event here?